src/utilities/theme/context.tsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import {Theme} from './types';
 
-export interface ThemeProviderContextType {
+export interface ThemeContextType {
   logo: Theme['logo'] | null;
 }
 
-export const ThemeProviderContext = React.createContext<
```

Comment:
I'm a little conflicted about this. The reason why I could see this staying `ThemeProviderContext` is that it's a component context that matches the name of the component; `ThemeProviderContext`, `ResourceListContext`, `FrameContext`, etc. What are your thoughts on this?

Reply:
Per conversation:

IMO The most important part of these contexts is where they get called, in this case components that use `useTheme()`, not where the value came from. it seems odd for the return type of `useTheme` to be `ThemeProviderContext` - why do you care about the component that provided this context, you only care about the value? `ThemeContext` reflects that. In tests we never call the `ThemeProvider` component so trying to hint at the name of the component that provides the context value is misleading anyway.

I'd like to follow the road we started on by splitting the AppProvider up into multiple contexts where one component doesn't necessitate a single context, such a binding feels restrictive.